### ISSUE TITLE
Create LiteDRAM Avalon frontend.

### ISSUE THREAD

**Issue by enjoy-digital** [enjoy-digital/litex_mister_test#6]
Required by https://github.com/enjoy-digital/litex_mister_test/issues/3 and will be useful to LiteX in general.

**Comment by hansfbaier** [comment]
Yes I really thought that would come in handy. Maybe even I could do that.
